code/src/components/app-table/AppTable.tsx

```diff
@@ -22,30 +24,39 @@ type AppTableProps<T> = {
   headItems: string[];
   renderBodyItem: (item: T) => ReactNode;
   renderHeadItem: (item: string) => ReactNode;
+  fallback?: ReactNode;
 };
```

Comment:
I suggest moving this type to `AppTable.types.ts`

Reply:
Moved